nvflare/app_common/widgets/streaming.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2021-2023, NVIDIA CORPORATION.  All rights reserved.
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
```

Comment:
For `AnalyticsSender` I noticed we deprecate some methods here.

Maybe we can write a decorator of "deprecated" or similar.
And we can pass those deprecation information to that decorator, and it will prints out automatically when user use that method.

For example:

```
@deprecated("This method is deprecated, please use :py:class:`TBWriter <nvflare.app_opt.tracking.tb.tb_writer.TBWriter>` instead")
def add_scalar(self, tag: str, scalar: float, global_step: Optional[int] = None, **kwargs):
      """Sends a scalar."""
```